src/legacy/core_plugins/ui_metric/README.md

```diff
@@ -58,4 +72,6 @@ These saved objects are automatically consumed by the stats API and surfaced und
 ```
 
 By storing these metrics and their counts as key-value pairs, we can add more metrics without having
-to worry about exceeding the 1000-field soft limit in Elasticsearch.
\ No newline at end of file
+to worry about exceeding the 1000-field soft limit in Elasticsearch.
+
+// TODO: Clarify that colons are NOT allowed in metric names or app types possibly reject request in trackUiMetric; disallow commas too?
```

Comment:
I think this todo is no longer needed as you have already addressed it on line 37